Scripts/SL-PlayerProfiles.lua

```diff
@@ -49,6 +49,9 @@ local permitted_profile_settings = {
 	Pacemaker            = "boolean",
 	MissBecauseHeld      = "boolean",
 	NPSGraphAtTop        = "boolean",
+	ErrorBar             = "string",
+	ErrorBarMoveUp       = "boolean",
```

Comment:
Should we just call this ErrorBarUp ( to make is similar to MeasureCountUp). Actually sounds nicer to rename MeasureCounterUp to MeasureCounterMoveUp (same with Left).

Reply:
done!